scripts/plotting/zonal_mean.py

```diff
@@ -86,7 +86,11 @@ def zonal_mean(adfobj):
     # -- So check for it, and default to png
     basic_info_dict = adfobj.read_config_var("diag_basic_info")
     plot_type = basic_info_dict.get('plot_type', 'png')
-    print(f"NOTE: Plot type is set to {plot_type}")
+    print(f"\t NOTE: Plot type is set to {plot_type}")
+
+    # check if existing plots need to be redone
+    redo_plot = adfobj.get_basic_info('redo_plot')
+    print(f"\t NOTE: redo_plot is set to {redo_plot}")
```

Comment:
@nusbaume: I don't think we want to do it that way because there will be case where the user will do the zonal mean without doing the other types of plots. SO I suggest we will leave this print statement there for now and we work on a verbose flag that will control the level of output.